index.js

```diff
@@ -1158,268 +1163,154 @@ const createJavaBom = async (path, options) => {
       path,
       (options.multiProject ? "**/" : "") + "build.gradle*"
     );
+    let allProjects = [];
+    const rootDependsOn = [];
+    // Execute gradle properties
+    if (gradleFiles && gradleFiles.length) {
+      let retMap = utils.executeGradleProperties(path, null, null);
+      const allProjectsStr = retMap.projects || [];
+      let rootProject = retMap.rootProject;
+      if (rootProject) {
+        parentComponent = {
+          name: rootProject,
+          type: "maven",
+          qualifiers: { type: "jar" },
+          ...(retMap.metadata || {})
+        };
+      }
+      // Get the sub-project properties and set the root dependencies
+      if (allProjectsStr && allProjectsStr.length) {
+        for (let spstr of allProjectsStr) {
+          retMap = utils.executeGradleProperties(path, null, spstr);
+          let rootSubProject = retMap.rootProject;
+          if (rootSubProject) {
+            const rootSubProjectObj = {
+              name: rootSubProject,
+              type: "maven",
```

Comment:
"maven" intended?

Reply:
Good catch! This was getting overwritten to library few lines later but will remove the hack.